lib/Tvm.js

```diff
@@ -44,12 +45,13 @@ class Tvm {
   constructor () {
     this._validationSchema = {
       /// deployment params - should not be setable by request - deployed as final
-      expirationDuration: joi.number().integer().min(900).max(18000).required(),
+      expirationDuration: joi.number().integer().min(600).max(18000).required(),
```

Comment:
Can we change this to be cosmos specific ? I don't remember exactly but maybe 900 was a minimum for aws s3 or azure blob, if not then it's fine

Reply:
Let me check this. I don't think 10 mins would hit azure blob minimum, but I will reconfirm this.

Reply:
sure also need to check for aws s3